content/kots-cli/admin-console/generate-manifests.md

```diff
@@ -0,0 +1,29 @@
+---
+date: 2021-07-29
+linktitle: "generate-manifests"
+title: kots admin-console generate-manifests
+weight: 90130
+---
+
+If you’d rather use kubectl or another workflow to deploy to a cluster, run `kots admin-console generate-manifests` to create a directory on the workstation containing the KOTS Admin Console manifests. This workflow enables additional customization of the Admin Console before deploying.
+
+### Usage
+```bash
+kubectl kots admin-console generate-manifests [flags]
+```
+* _Provide `[flags]` according to the table below_
```

Comment:
I think we should match the format of other CLI commands; also are global flags supported? 
Replacement could be:
This command supports all [global flags](/kots-cli/global-flags/) and also:

Reply:
That line was actually copied from the `kots pull` docs but I'm fine with removing it. I didn't include the part about global flags because they're not supported currently. But definitely open to matching the format of other docs!

I'm not sure `kots pull` really supports global flags either (even though it says so). Looking at the code, that CLI command doesn't seem to read any of the global flags.

Reply:
Totally makes sense; I wasn't sure about the globals part at all but figured to include just in case; could likely be adjusted to:
This command supports the following flags: